setup.sh

```diff
@@ -20,7 +20,7 @@ osx() {
 
 install_apt() {
     sudo apt-get update || true
-    sudo apt-get install -y git gdb gdbserver python3-dev python3-pip python3-setuptools libglib2.0-dev libc6-dbg
+    sudo apt-get install -y git gdb gdb-multiarch gdbserver python3-dev python3-pip python3-setuptools libglib2.0-dev libc6-dbg qemu-system-x86 qemu-system-arm
```

Comment:
The `qemu-system-x86 qemu-system-arm` are not direct Pwndbg dependencies; shall we only install those in setup-dev.sh?

Reply:
You're right, will do. `gdb-multiarch` also isn't technically a pwndbg dependency either, so I'll move it as well.